src/web/app/page.component.html

```diff
@@ -5,7 +5,7 @@
   <button class="navbar-toggler d-lg-none" type="button" (click)="toggleCollapse()" [attr.aria-expanded]="!isCollapsed" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
-  <div class="collapse navbar-collapse" [ngbCollapse]="isCollapsed">
+  <div class="navbar-collapse" [ngStyle]="{display: isCollapsed ? 'none' : 'block'}">
```

Comment:
Changed this because the collapse class sets `display: none` by default, which conflicts with the change. This is basically the same end result.